util/analogsim_asic.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+#  Change lines:
+#     inout CC1, // Manual Pad
+#     inout CC2, // Manual Pad
+#  To:
+#     `INOUT_AI CC1, // Manual Pad
+#     `INOUT_AI CC2, // Manual Pad
+#
+#  Change lines:
+#     inout IOA2, // MIO Pad 2
+#     inout IOA3, // MIO Pad 3
+#  To:
+#     `INOUT_AO IOA2, // MIO Pad 2
+#     `INOUT_AO IOA3, // MIO Pad 3
+#
+#  Change lines:
+#         CC2,
+#         CC1,
+#  To:
+#         `CC2_A,
+#         `CC1_A,
+#
+#  Change lines:
+#        IOA3,
+#        IOA2,
+#  To:
+#        `IOA3_A,
+#        `IOA2_A,
+
+import re
+
+input_file = "top_earlgrey/rtl/autogen/chip_earlgrey_asic.sv"
+
+replacements = [('inout CC1', '`INOUT_AI CC1'),
```

Comment:
I can not do it in your .hjson since I can not resolve the complexity of it.
I took the simplest approach to modify 8 lines of @msfschaffner  output file.
As I mentioned in the past, doing it inside the .hjson is more complex then 15 line python script.
@tjaychen Can you approve it for now,  and we can address it latter.